code/game/response_team.dm

```diff
@@ -99,7 +99,7 @@ var/can_call_ert
 	while(send_emergency_team == 0) // There is no ERT at the time.
 		var/decl/security_state/security_state = decls_repository.get_decl(GLOB.using_map.security_state)
 		var/index = list_find(security_state.all_security_levels, security_state.current_security_level)
-		ert_base_chance += 2**index
+		ert_base_chance += index
```

Comment:
Why the change 'ere?

Reply:
This was designed for 3 usable alert levels, but we have 6

Reply:
For reference, if the facility is at Code Black for 30 minutes, that ups the base chance by 50%